plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/types.ts

```diff
@@ -0,0 +1,6 @@
+export type ProductTypeSlug = 'single-product' | string;
```

Comment:
What does `single-product` mean here? Should it be used as a generic product type when we don't know it?